Gemfile.lock

```diff
@@ -44,7 +44,7 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.6.0)
+    activeadmin (2.6.1)
```

Comment:
### [activeadmin](https://activeadmin.info)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.6.0 → 2.6.1

[_~~change-log~~, [source-code](http://github.com/activeadmin/activeadmin)_]





<details>
<summary>Commits</summary>

A change of **94** commits. See the full changes on [the compare page](https://github.com/activeadmin/activeadmin/compare/v2.6.0...v2.6.1).

These are the first 10 commits:
- (d24f7f4) [Bump rubocop-rspec from 1.37.0 to 1.37.1 (#5977)](https://github.com/activeadmin/activeadmin/commit/d24f7f45fa676642ecc7191b041946fe41ed9cf5)
- (6874da4) [Bump rails from 5.2.4 to 5.2.4.1 in /gemfiles/rails_52 (#597…](https://github.com/activeadmin/activeadmin/commit/6874da420e46578c2d2ad428b7914b0ea54666d5)
- (b24c57a) [Bump rails from 6.0.2 to 6.0.2.1 in /gemfiles/rails_60_turbo…](https://github.com/activeadmin/activeadmin/commit/b24c57a05e50467806093590f773a81d34011ec3)
- (068bf75) [[Security] Bump rack from 2.0.7 to 2.0.8 (#5981)](https://github.com/activeadmin/activeadmin/commit/068bf75e385262c5d76d0213f7e370177c4c9d03)
- (63212c8) [Bump rails from 6.0.2 to 6.0.2.1 (#5982)](https://github.com/activeadmin/activeadmin/commit/63212c844584728c001818c57051ed104d96d437)
- (637af06) [Bump sqlite3 from 1.4.1 to 1.4.2 in /gemfiles/rails_60_turbo…](https://github.com/activeadmin/activeadmin/commit/637af06dfe85a6cf198217932f8bcdd68a311bb6)
- (8a92714) [Bump rubocop from 0.77.0 to 0.78.0 (#5987)](https://github.com/activeadmin/activeadmin/commit/8a92714721ba5cba978fc59afa612a2b32adb74c)
- (8eb7196) [Bump sqlite3 from 1.4.1 to 1.4.2 (#5984)](https://github.com/activeadmin/activeadmin/commit/8eb7196bb2eb207db41192942273f561026c1dd2)
- (953e135) [Bump sqlite3 from 1.4.1 to 1.4.2 in /gemfiles/rails_52 (#598…](https://github.com/activeadmin/activeadmin/commit/953e1359ba6d0f648538a0a6724f4722b802721c)
- (bb31dca) [Bump capybara from 3.29.0 to 3.30.0 in /gemfiles/rails_60_tu…](https://github.com/activeadmin/activeadmin/commit/bb31dca242f4f94141250873079009db8839cbe8)


</details>